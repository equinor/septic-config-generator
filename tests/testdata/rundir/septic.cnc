// Messages that should be reported by check
UNABLE to connect to server SomeServer
ERROR adding Item: SomeTag
