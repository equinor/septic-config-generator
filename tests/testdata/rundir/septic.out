// A selection of messages that should (?) be reported by check
QP WARNING = 
SQP WARNING =
WARNING, avoid 
WARNING, duplicate Xvr name found
***WARNING*** : 
Thu Jun  1 08:08:09 2023 N = 10: Calc WARNING
WARNING: Type mismatch between SopcEvr: TestTvr and Xvr: TestTvr" 
** WARNING: MULTIPLE models for SmpcAppl
10: MVR TestMvr VALIDATION WARNING: 
** ILLEGAL object type
** ILLEGAL keyword

10: FMU error: Could not get initial inputs
10: FMU error: Could not get outputs 
ERROR: Backup cnfg save failed at startup 
** 'cvpred()' CALC ERROR: missing parameter
Thu Jun  1 08:08:09 2023 N = 10: Calc ERROR TestCalc.Alg=
** CALC ERROR for CalcPvr 'TestPvr.member'
** CNFG ERROR missing/wring list for
** FATAL ERROR, program will terminate: Missing model for SmpcAppl **

** MISSING appl for SopcProc: TestSopcProc  **
 ** MISSING appl for OPCProc: TestOPCProc  **
** MISSING CalcModl for CalcPvr TestPvr **
MISSING 'models' directory

INFO: Validation tag is missing from SopcSampleTvrs!
Thu Jun  1 08:08:09 2023 INFO: MODL message with unknown key at end

No Xvr match for Pvr TestPvr
No matching XVR found for SopcEvr: TestEvr